SQF/dayz_code/stringtable.xml

```diff
@@ -5462,7 +5462,7 @@
 		</Key>
 		<Key ID="STR_ITEMSODA_FULL_CLEAN_CODE_NAME_11">
 			<English>Beer</English>
-			<German>Dose (Bier)</German>
+			<German>Bier</German>
```

Comment:
Please keep the "Dose" prefix. It looks better in the trader menu and it is for all soda cans.